internal/dao/ingestion_task_test.go

```
@@ -104,129 +103,169 @@ func TestIngestionTaskDAODocumentIDIsUniqueAtDBLevel(t *testing.T) {
-		t.Fatalf("create task %s: %v", id, err)
+		t.Fatalf("create scheduled task: %v", err)
+	}
+
+	const callbackName = "test:claim-ingestion-task-before-delete"
+	if err := db.Callback().Delete().Before("gorm:delete").Register(callbackName, func(tx *gorm.DB) {
+		if tx.Statement.Table != task.TableName() {
+			return
+		}
+		if err := tx.Session(&gorm.Session{NewDB: true}).Model(&entity.IngestionTask{}).Where("id = ?", task.ID).
+			Update("status", common.RUNNING).Error; err != nil {
+			tx.AddError(err)
```

golangci-lint (2.12.2)

[error] Error return value of `tx.AddError` is not checked

(errcheck)